app/controllers/media_controller.rb

```diff
@@ -0,0 +1,7 @@
+class MediaController < ApplicationController
+  def index
+    @books = Book.all.order(:upvotes).reverse.take(10)
```

Comment:
This could be a good place to use a model scope.

Example in controller:
`Book.best(10)`

Example in the Book model:
`scope :best, -> (total) { order('upvotes DESC').limit(total)}`